README.md

```diff
@@ -55,6 +55,8 @@ From 2.1.0 onward we also provide support for akka-typed. This is done as a sepa
 * Include:
   `libraryDependencies += "io.altoo" %% "akka-kryo-serialization-typed" % "2.2.0"`
 
+Version 2.2.0 drops java 8 support in favor of optimizations using ByteBuffer. 
```

Comment:
We should note that 2.2.0 requires JDK 11 or higher

Reply:
absolutely... adapted